<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeInputDialog</class>
 <widget class="QWidget" name="DateTimeInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>179</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>179</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>179</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Date/Time</string>
  </property>
  <widget class="QCalendarWidget" name="m_calendar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>171</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="m_timeLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="verticalSpacing">
     <number>26</number>
    </property>
    <item row="0" column="1">
     <widget class="QTimeEdit" name="m_currentTimeDisplay">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="displayFormat">
       <string>hh:mm:ss AP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_selectedTimeLabel">
      <property name="text">
       <string>Selected Time:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTimeEdit" name="m_selectedTimeInput">
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>hh:mm:ss AP</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="m_currentTimeLabel">
      <property name="text">
       <string>Current Time:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
